index.html

```diff
@@ -0,0 +1,253 @@
+            </nav>
+            <div id="myNav" class="overlay">
+                <a class="closebtn"> &times; </a>
+                <div class="overlay-content">
+                    <a class="pop-nav" href="#portfolio">Portfolio</a>
+                    <a class="pop-nav" href="#about">About</a>
+                    <a class="pop-nav" href="#contact">Contact</a>
+                </div>
+            </div>
+        </div>
+        <nav class="web-nav">
+            <div class="logo">
+                <a class="Mylogo" href="">My Logo</a>
+            </div>
+            <ul>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="content">
+        <h1 class="intro">I'm Josue Rushanika Nice to see you!</h1>
+        <p class="intro1">
+            I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+        <h3 class="intro2">LET'S CONNECT</h3>
+        <ul class="social">
+            <li class="social-icon twitter">
+                
+                <a href="https://twitter.com/josuerushanika"><img src="images/twitter.png" alt="twitter"></a>
+            </li>
+            <li class="social-icon linkedin">
+                <a href="https://www.linkedin.com/in/josue-rushanika-336049b3/"><img src="images/Linkedin.png" alt="linkedin"></a>
+            </li>
+            <li class="social-icon">
+                <a href="#"><img src="images/m.png" alt="m"></a> 
+            </li>
+            <li class="social-icon github">
+                <a href="https://github.com/josuerushanika"><img src="images/github.png" alt="github"></a>
+            </li>
+            <li class="social-icon fingers">
+                <a href="#"><img src="images/fingers.png" alt="fingers"></a>
+            </li>
+        </ul>
+    </section>
+
+    <section class="works-section container">
+    <article class="card">
+        <div class="card-image">
+           <img src="images/portfolio-1.svg" alt="Portfolio">
+        </div>
+        <div class="card-content">
+         <h2 class="card-title">Tonic</h2>
+         <ul class="card-info">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+         </ul>
+         <P class="card-description">
+           A daily selectin of privalety personalized reads; no accounts
+           or sign-ups required.
+         </P>
+         <ul class="card-tags">
+            <li>html</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+         </ul>
+         <button class="primary-button card-button" type="button">
+            see Project
+         </button>
+        </div>
+    </article>
```

Comment:
Your design 👇 
![image](https://user-images.githubusercontent.com/78251927/205442135-51e8d921-3b64-4f5c-91af-0793aa77d4c6.png)

Figma design 👇 
![image](https://user-images.githubusercontent.com/78251927/205442208-1b60f908-bfdd-42cd-9da3-669facb89fba.png)

- [ ] Please kindly use the same font as [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) in this section as mentioned in the[ project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md).